exampleSite/config.toml

```diff
@@ -28,6 +28,7 @@ googleAnalytics = "XXX"
     shareTwitter = true
     shareFacebook = true
     shareGooglePlus = true
+    shareLinkedIn = true
```

Comment:
This has to default to `false` so that we do not enable this for existing users by default